versioned_docs/version-2022sp/lecture1.md

```diff
@@ -3,4 +3,532 @@ id: lecture1
 title: Lecture 1
 ---
 
-To be added
+Lecture Slides(to be added later)
+
+No homework this week!
+
+## JavaScript
+
+### What is JavaScript
+
+- JavaScript is the de facto language of the web
+- Commonly used in conjunction with HTML/CSS
+- Became really popular for powering client-side logic through AJAX
+  - Previously, languages like PHP had to communicate with the server before
+    coming back with a response
+- These days, JavaScript is everywhere!
+
+:::note
+Java is to JavaScript as car is to carpet. They are very different languages!
+:::
+
+### Basic JavaScript Syntax
+
+#### Variables
+
+There are three ways to create variables in JS:
+
+1. ~~`var x = 5`~~
+2. `let x = 5`
+3. **`const x = 5`**
```

Comment:
Can we clarify on what const actually does? I think immutability is the wrong word, it just prevents reassignment.

Reply:
Changed to mean non-reassignable based on our language today in class